SPECS/cronie/cronie.spec

```diff
@@ -0,0 +1,82 @@
+Summary:	Cron Daemon
+Name:		cronie
+Version:	1.5.0
+Release:	1%{?dist}
+License:	GPLv2+ and MIT and BSD and ISC
+URL:		https://fedorahosted.org/cronie
+Source0:	https://fedorahosted.org/releases/c/r/cronie/%{name}-%{version}.tar.gz
+Group:		System Environment/Base
+Vendor:		VMware, Inc.
+Distribution:	Photon
+BuildRequires:	libselinux-devel
+BuildRequires:	Linux-PAM
+Requires:	libselinux
+Requires:	Linux-PAM
+%description
+Cronie contains the standard UNIX daemon crond that runs specified programs at
+scheduled times and related tools. It is based on the original cron and
+has security and configuration enhancements like the ability to use pam and
+SELinux.
+%prep
+%setup -q
+%build
+autoreconf
+./configure \
+	--prefix=%{_prefix} \
+	--with-pam	\
+	--with-selinux \
```

Comment:
we may or may not have SELinux enabled? I don't know how this going to effect this package?